src/views/InvitationSection.vue

```diff
@@ -28,15 +28,15 @@
         <template #header>
           <span class="p-input-icon-right search-field">
             <i class="pi pi-search" />
-            <InputText placeholder="Введіть електронну адресу" />
+            <InputText @input="searchEmail" v-model="searchQuery" placeholder="Введіть електронну адресу" />
```

Comment:
There can be some performance issues, because of the requests, that will go to the server everytime you type a letter, it's better to make some delay inside 'searchEmail' method